src/build.d

```diff
@@ -366,6 +365,16 @@ and afterwards builds the DMD compiler.
 */
 auto buildDMD()
 {
+    version(Windows)
+    {
+        immutable model = detectModel;
```

Comment:
BTW as `detectModel` is now called 3x we might want to cache it in the future, but I think the cost of one `uname` is insignificant to invoking DMD many times.

Reply:
Yeah, I'm not too happy about that either.  But, if I'm permitted to have my druthers, after this is working for Windows, added to the CIs, and given a little field testing, I would like to do a major refactor of the code so it no longer resembles a makefile, but rather more of an idiomatic D program.  We'll see, and I'm sure there will be differences of opinions, but I'm sure we'll get something quite nice in the end.